doc/api/next_api_changes/2019-07-18-SL.rst

```diff
@@ -1,4 +1,26 @@
 Reduced default value of :rc:`axes.formatter.limits`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Changed the default value of :rc:`axes.formatter.limits` from -7, 7 to -5, 6 for better readability.
+Changed the default value of :rc:`axes.formatter.limits` from -7, 7 to
+-5, 6 for better readability.
+
+.. plot::
+
+   import matplotlib.pyplot as plt
+   import numpy as np
+
+   fig, [ax_old, ax_new] = plt.subplots(1, 2, constrained_layout=True)
```

Comment:
```suggestion
   fig, (ax_old, ax_new) = plt.subplots(1, 2, constrained_layout=True)
```

We tend to use a tuple not a list in this pattern (Code search: 9x list, 90x tuple). Not that I have a strong feeling either way, but ticking to one pattern feels better.